src/HttpClient/Plugin/Authentication/GceServiceAccount.php

```diff
@@ -61,7 +62,7 @@ public function isAvailable(): bool
             $this->authClient()->get('');
 
             return true;
-        } catch (Exception $e) {
+        } catch (NetworkExceptionInterface $e) {
```

Comment:
@mxr576 ,
With `NetworkExceptionInterface` I am not able to access the `admin/config/apigee-edge/settings` page for credential settings.

Screenshot :
![image](https://github.com/apigee/apigee-client-php/assets/75600200/6aa5cdfd-95cd-41be-b607-633aca63d00c)

Reply:
Thanks for checking, I think I have found the root cause of that: https://github.com/apigee/apigee-edge-drupal/pull/1019

Reply:
@mxr576 Thankyou